src/main/java/com/example/petbuddybackend/controller/CareController.java

```diff
@@ -104,9 +104,13 @@ public CareDTO updateCare(
     @PostMapping("/{careId}/accept")
     @Operation(
             summary = "Accept a care",
-            description = "Accepts a care. " +
-                    "The care must be in the PENDING status for caretaker and ACCEPT status for client to accept." +
-                    "When successful both statuses changes to AWAITING_PAYMENT."
+            description = """
+                    Caretaker accepts a reservation that client sends and client accepts the care that was edited by
+                    caretaker.
+                    
+                    The care must be in the PENDING status for caretaker and ACCEPT status for client to accept.
+                    When successful both statuses changes to AWAITING_PAYMENT.
```

Comment:
And care must be in PENDING status for client and ACCEPT status for caretaker to accept also